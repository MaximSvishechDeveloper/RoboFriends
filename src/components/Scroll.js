import React from "react";

const Scroll = (props) => {
  return (
    <div
      style={{ overflowY: "auto", border: "1px solid black", height:"auto" }}
    >
      {props.children}
    </div>
  );
};

export default Scroll;
