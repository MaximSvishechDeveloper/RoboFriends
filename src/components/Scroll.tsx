import React, { ReactNode } from "react";

interface ScrollProps {
  children: ReactNode;
}

const Scroll: React.FC<ScrollProps> = ({ children }) => {
  return (
    <div
      style={{ overflowY: "auto", border: "1px solid black", height: "auto" }}
    >
      {children}
    </div>
  );
};

export default Scroll;
