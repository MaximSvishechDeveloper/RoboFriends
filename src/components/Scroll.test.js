import { shallow } from "enzyme";
import Scroll from "./Scroll";
import React from "react";

it("renders without crashing", () => {
  expect(shallow(<Scroll />)).toMatchSnapshot();
});
