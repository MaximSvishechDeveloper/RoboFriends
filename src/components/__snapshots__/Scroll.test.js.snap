// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  style={
    Object {
      "border": "1px solid black",
      "height": "auto",
      "overflowY": "auto",
    }
  }
/>
`;
